### DISCUSSION TITLE
Lint specific rule + auto-add ignore comments

### DISCUSSION THREAD

**Discussion by Faithfinder** [eslint/eslint#18401] [Ideas]
Started a new job recently and have opened the new to me world of Python. Where I met the local counterpart to Biome (With Eslint's counterpart being Flake8, I suppose?) - [Ruff](https://docs.astral.sh/ruff/). And it has a couple of very nice capabilities that work together really well:

* `ruff check --select %RULE%` would lint the codebase using that rule only
* `ruff check --add-noqa` would automatically add ignore comments to every violation

Combined together, it means that you can easily evaluate a new rule for your codebase, and either auto-fix that exact rule (safe or unsafe fixes), or ignore all existing occurrences, all without messing with the config. I loved the experience and would love something like that in JS world

**Comment by nzakas** [comment]
Those are interesting ideas, thanks for sharing. I think `--select` is something that would be fairly easy to implement. If you open up a feature request and describe a bit more about it, we can take a look.

The other I think is probably a bit more controversial, but worth looking into as well. Same thing -- if you open a feature request and describe a bit more about how you use it, that would be helpful.
